### DISCUSSION TITLE
Is there a way to set the position to the top of the page?

### DISCUSSION THREAD

**Discussion by ke-helle** [usernotnull/tall-toasts#259] [Q&A] [answered]
Due to my layout it would be prefered to change the defualt bottom-right position to top-right. Is this posible?

**Comment by usernotnull** [comment, marked as answer]
Sure @webstudios1! 

Just edit the layout after you publish the views, as described in the docs. 

You can literally change the whole view.

**Reply by AleJavierHY** [reply]
To change the position up I followed those steps, but the new notifications appear below the old ones and when they accumulate they are lost from sight.
Is there any way to change this behavior?

**Reply by usernotnull** [reply]
> To change the position up I followed those steps, but the new notifications appear below the old ones and when they accumulate they are lost from sight.
> Is there any way to change this behavior?

The behavior is based on the frontend layout only. Practice your tailwindcss skills there is a way :)
